modules/volunteers/admin/sitesupport/giftTable.js

```diff
@@ -260,6 +244,43 @@ export default {
         return [u.length, nonGroup];
       }
       return [0, 1];
+    },
+    sortTable(column) {
+      if (column.unsortable) {
+        return;
+      }
+      if (column.value === this.sortColumn) {
+        this.decendingSort = !this.decendingSort;
+      } else {
+        this.decendingSort = false;
+      }
+      const sortDirection = this.decendingSort ? -1.0 : 1.0;
+      this.sortColumn = column.value;
+
+      this.records.sort((a,b) => {
+        let result = (parseFloat(a[column.value]) - parseFloat(b[column.value])) * sortDirection;
+        if (column.value === 'remaining') {
+          result = ((a.inventory - a.claimed) - (b.inventory - b.claimed)) * sortDirection;
+        } else if (column.type === String) {
+          result = a[column.value].localeCompare(b[column.value]) * sortDirection;
+        }
+
+        if (result === 0.0) {
```

Comment:
Curious whether this may give us trouble down the road due to floating point precision. I guess if it comes up, maybe we address it at that time. I think some eventual options might be things like `Math.floor()` or `Number.toPrecision()`